S100-Shop-Facade-Open-Api-Sepc-v0.1.yaml

```diff
@@ -0,0 +1,1145 @@
+          headers:
+            X-Correlation-ID:
+              description: GUID for the request for logging/tracing
+              schema:
+                type: string
+          content: {}
+        "403":
+          description: Forbidden - you have no permission to use this API.
+          headers:
+            X-Correlation-ID:
+              description: GUID for the request for logging/tracing
+              schema:
+                type: string
+        "404":
+          description: Licence not found.
+          headers:
+            X-Correlation-ID:
+              description: GUID for the request for logging/tracing
+              schema:
+                type: string
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/errorDescription'
+              example:
+                correlationId: 6c254109-8a0f-4446-8649-721775ba9de2
+                errors:
+                - source: licenceId
+                  description: Licence Not Found.
+        "429":
+          description: Too Many Requests.
+          headers:
+            X-Correlation-ID:
+              description: GUID for the request for logging/tracing
+              schema:
+                type: string
+        "500":
+          description: Internal Server Error.
+          headers:
+            X-Correlation-ID:
+              description: GUID for the request for logging/tracing
+              schema:
+                type: string
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/exceptionDescription'
+                
+  /holdings/{licenceId}/s100:
+    get:
+      description: Get holdings for the requested licence.
+      operationId: getHoldings
+      tags:
+        -  ""
+      parameters:
+      - name: licenceId
+        in: path
+        description: Licence Id. It must be an integer value and greater than zero.
+        required: true
+        schema:
+          type: integer
+          example: 12345678
+      responses:
+        "200":
+          description: OK - Returns holdings available for the licence.
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  productCode:
+                    type: string
+                    description: ProductCode define the type of product.
+                    example: "P1231"
+                  productTitle:
+                    type: string
+                    description: productTitle define the title of product.
+                    example: "P1231"
+                  expiryDate:
+                    type: string
+                    description: expiryDate of the product.
+                    example: "2026-01-31T23:59:00Z"
+                  cells:
+                    type: array
+                    items:
+                      $ref: "#/components/schemas/cells"
```

Comment:
Unless they are used elsewhere, would it be easier to maintain if the array items are inline?

Reply:
Updated.